shared/common-adapters/list-item2.tsx

```diff
@@ -36,9 +37,11 @@ type Props = {
 
 const ListItem = (props: Props) => (
   <Kb.ClickableBox
-    onClick={props.onClick}
+    onClick={props.onClick || (props.onMouseDown ? () => {} : undefined)} // cause click styles if just onMouseDown is defined
```

Comment:
I don't understand this comment

Reply:
Ok now I do but possibly it could be better worded. Maybe "make sure the box gets styled clickable if just..."?

Reply:
I'll apply this to the integration pr